WcaOnRails/config/locales/en.yml

```diff
@@ -995,6 +995,7 @@ en:
       national_records: "History of National Records"
   posts:
     confirm_delete_post: "Are you sure you want to delete this post? This action is not reversible."
+    announcement_info: "Announced by %{author_name} on %{date_time}"
```

Comment:
That's interesting. Without the call to wca_local_time, how does this work? Does it end up in server time?